scripts/globals/summon.lua

```diff
@@ -18,9 +18,28 @@ function AvatarPhysicalMove(avatar, target, skill, numberofhits, accmod, dmgmod,
     local dmg = avatar:getWeaponDmg()
     local minFstr, maxFstr = avatarFSTR(avatar:getStat(tpz.mod.STR), target:getStat(tpz.mod.VIT))
     local ratio = avatar:getStat(tpz.mod.ATT) / target:getStat(tpz.mod.DEF)
+    local lvluser = avatar:getMainLvl()
+    local lvltarget = target:getMainLvl()
+    local firstacc = acc + 100
+
+-- formula should be 75 + floor( (Accuracy - Evasion)÷2 ) - 2×(dLVL)
+-- while level correction does not apply to damage, it doesnt appear to correspond to hit rate.
+    local hitdiff = 0
+    local hitrate = 75
+    local firsthit = 0
+
+    if (lvluser > lvltarget) then
+        hitdiff = hitrate + math.floor(acc + bonusacc - eva) / 2 
+        firsthit = hitrate + math.floor(firstacc + bonusacc - eva) / 2 
+    else 
+        hitdiff = hitrate + math.floor((acc + bonusacc - eva) /2) + ((lvltarget - lvluser) *2 )
+        firsthit = hitrate + math.floor((firstacc + bonusacc - eva) /2) + ((lvltarget - lvluser) * 2)
+    end
 
-    -- Note: Avatars do not have any level correction. This is why they are so good on Wyrms! // https://kegsay.livejournal.com/tag/smn!
```

Comment:
RIP this link, only part of it is still available on wayback machine:
https://web.archive.org/web/20180926041335/https://kegsay.livejournal.com/#item3946

Don't know if it's worth keeping any of this information around?

Reply:
the link still works, it just actually needs the !

Reply:
I've also made sure the wayback machine now has the most current version, just in caser the original is someday not available.

https://web.archive.org/web/20200905213200if_/https://kegsay.livejournal.com/tag/smn!